build.gradle

```diff
@@ -1,16 +1,17 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
+apply plugin: 'com.jfrog.bintray'
+apply plugin: 'maven-publish'
+
 buildscript {
     ext.kotlin_version = "1.3.72"
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.0.0"
+        classpath "com.android.tools.build:gradle:4.0.1"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.3'
```

Comment:
I recommend you to remark this because latest version of `gradle-bintray-plugin` causes `NullPointerException`. (I suffered because of it 😭 )
```
// Do not update before
// https://github.com/bintray/gradle-bintray-plugin/issues/267 is resolved.
```

Reply:
Thank you 😭😭